### DISCUSSION TITLE
Z-Wave JS is up and running, but I can't see devices in Domoticz, how to make that happen?

### DISCUSSION THREAD

**Discussion by Pepito-design** [community/community#86706] [Programming Help]
### Select Topic Area

Question

### Body

Hi,
was able to make Z-Wave JS work on a RPI 4 with latest 64 Bit Raspian OS ( new installation ).
Also brand new Domoticz installation on it.
I can see, that  Z-Wave JS has access via Z-Wave Stick
Bus 001 Device 004: ID 0658:0200 Sigma Designs, Inc. Aeotec Z-Stick Gen5 (ZW090) - UZB
to my 6 Danfoss thermostats.
It's possible to set and get values ( Z-Wave JS web based UI ).
In the past it was used with old Z-Wave integration and worked fine for some years. Then the update and I had to
start from scratch.
Now I'm hanging to glue Z-Wave JS and Domoticz together.
Is there a step by step instruction available?
First to add Hardware,
then Devices .... 
Thanks in advance for every hint.
Peter

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
